docs/api.json

```diff
@@ -0,0 +1,1070 @@
+      "tuesday": "Boolean",
+      "wednesday": "Boolean",
+      "thursday": "Boolean",
+      "friday": "Boolean",
+      "saturday": "Boolean",
+      "sunday": "Boolean",
+      "deleted": "Boolean"
+    },
+    "visit": {
+      "id": "Number",
+      "activity": "String",
+      "visitDate": "String"
+    },
+    "outreach": {
+      "id": "Number",
+      "organisationName": "String",
+      "userOwner": "String",
+      "outreachCampaignType": "String",
+      "outreachCampaignTarget": ["String"],
+      "createdAt": "String",
+      "modifiedAt": "String",
+      "meetings": ["types.meeting"]
+    },
+    "meeting": {
+      "id": "Number",
+      "meetingType": "String",
+      "outreachPartner": "String",
+      "createdAt": "String",
+      "modifiedAt": "String"
+    },
+    "volunteerLog": {
+      "id": "Number",
+      "volunteerActivity": "String",
+      "duration": "Number",
+      "loggedAt": "String"
+    }
+  },
+  "routes": {
+    "/sectors": {
+      "/": {
+        "GET": {
+          "description": "Retrieve all sectors",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin", "visitors", "volunteers", "volunteerAdmin"],
+          "scope": ["dropdown_options:read"],
+          "query": {},
+          "response": [{
+            "id": "String",
+            "sectorName": "String"
+          }]
+        },
+        "POST": {
+          "description": "Add a new sector",
+          "auth": true,
+          "intendedFor": ["twineAdmin"],
+          "scope": ["dropdown_options:write"],
+          "body": {
+            "sectorName": "String"
+          },
+          "response": [{
+            "id": "String",
+            "sectorName": "String"
+          }]
+        },
+        "PUT": {
+          "description": "Update a sector",
+          "auth": true,
+          "intendedFor": ["twineAdmin"],
+          "scope": ["dropdown_options:write"],
+          "body": {
+            "sectorName": "String"
+          },
+          "response": [{
+            "id": "String",
+            "sectorName": "String"
+          }]
+        },
+        "DELETE": {
+          "description": "Delete a sector",
+          "auth": true,
+          "intendedFor": ["twineAdmin"],
+          "scope": ["dropdown_options:delete"],
+          "query": {
+            "sectorName": "String"
+          },
+          "response": null
+        }
+      },
+      "/:id/organisations": {
+        "GET": {
+          "description": "Retrieve all organisations within a sector",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody"],
+          "scope": ["organisations_details-children:read"],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList": [{
+              "organisationDetails": "types.organisationDetails",
+              "childOrganisations": [{
```

Comment:
1. Many of these routes are designed to contain all the details for a relevant resource, with the client making use of the `fields` parameter for unwanted data. Whist creating the api it will become apparent if it's worth splitting this into its own route

2. Even though we no longer have a parent table in the database this relationship still exists, and is relevant to a number of places in the super admin dashboard for funding bodies such as subscriptions and data visualisation.